documentation/testing/test-strategy.md

```diff
@@ -89,54 +92,67 @@ be requested as appropriate.
 against deployed environments through a web-browser (such as Chrome, Safari, Firefox etc.) driven by web-drivers. This is the slowest type of (functional) automated
 testing are run later than unit and integration tests (see testing pyramid).
 
+These tests will run in testing environment after build stage in Testing environment. The testing environment will have pre loaded data to be used by tests. 
 These tests are the the responsibility of the test engineers to write and maintain in collaboration with the developers.
 
 ##### Technology Selected
 
-** *add technology here* **
+- [Playwright](https://playwright.dev/dotnet)
+- [Xunit](https://github.com/xunit)
+- [Specflow](https://specflow.org/)
+
+### Platform API Testing
 
-#### API Testing
 API tests are designed to test that the correct responses are received from the API when requests are made to it. These tests are run as part of the fuctional testing
 which runs as part of the pipeline with each deployment. These tests are written by collaboration of dev and test engineer.
+We have 2 types of API Tests. 
 
-##### Technology Selected
+#### API Integration Test
+Same as Web Application Integrations tests these tests are the responsibility of the developers to write and maintain in collaboration with the test engineers.
+In the pipeline the tests are kicked off in build stage after code changes in platform project. 
 
-** *add technology here* **
+##### Technology Selected
 
-#### Performance Testing
-Performance testing is used to evaluate how the service is able to cope in terms of its stability, responsiveness and usability when put under different simulated workloads.
-This ensures that the service copes with large amount of users accessing the service at one time.
+- [Moq](https://learn.microsoft.com/en-us/shows/visual-studio-toolbox/unit-testing-moq-framework)
 
+- [Xunit](https://github.com/xunit)
 
+#### API Functional Tests
+ These tests checks the functionality and behaviour of deployed APIs. These tests are kicked off after build stage using the test environment after code changes are made.
+These tests are the responsibility of the developers and Test Engineer(s) to write and maintain with collaboration. 
 ##### Technology Selected
 
-** *add technology here* **
+- [Xunit](https://github.com/xunit)
+- [Specflow](https://specflow.org/)
+
 
 #### Accessibility Testing
 Automated accessibility testing will be carried out using [Deque.axe.playwright](https://github.com/dequelabs/axe-core-nuget/blob/develop/packages/playwright/README.md)
 library to generate feedback on whether the service meets high level accessibility requirements in line with WCAG 2.2 AA.
-This will run as part of the pipeline in the later deployment steps from development to test and production.
+These tests will run every weekend in a separate pipeline using the testing environment. 
```

Comment:
Do we have a specific time they are expected to run, and a justification for that time? What impact, if any, could this have on the developers / users?